app/issues/IssuseAction.tsx

```diff
@@ -2,11 +2,13 @@ import { Button, Flex } from "@radix-ui/themes";
 import Link from "next/link";
 import React from "react";
 import IssuseStatusFilter from "./IssuseStatusFilter";
+import IssueAssigneeFilter from "./IssueAssigneeFilter";
 
 const IssuseAction = () => {
   return (
     <Flex justify="between">
```

Comment:
try to fix the CSS so that the filter can be near to each other.

Reply:
Done!